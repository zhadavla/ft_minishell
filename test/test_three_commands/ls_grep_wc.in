ls ~ -l -a | grep s | wc -l 