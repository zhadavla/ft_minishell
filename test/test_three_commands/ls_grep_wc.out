minishell$ ls ~ -l -a | grep s | wc -l 
65
minishell$ exit
